README.md

```diff
@@ -30,19 +31,33 @@ query and filter sample data via REST.
 
 #### Application information
 
-- Name: `loopback-example-model-relations`
-- Directory to contain the project: `loopback-example-model-relations`
-
 ```
 $ slc loopback loopback-example-model-relations
 ... # follow the prompts
+- Name: `loopback-example-model-relations`
+- Directory to contain the project: `loopback-example-model-relations`
```

Comment:
Should move these back to the top unless you want to copy all the prompts. I purposely list the app info first so they know what to do after. If you want to be more exact:

```
slc loopback loopback-example-model-relations
? What's the name of your application? loopback-example-model-relations
? Enter name of the directory to contain the project: loopback-example-model-relations
cd loopback-example-model-relations
```